plugins/luckgive/README.md

```diff
@@ -0,0 +1,30 @@
+# Cookies
+
+## Installation
+
+In the game, run `plugin/install luckgive`.
+
+It assumes that you are using the FS3Skills plugin and will not work without that plugin enabled.
+
+## Overview
+
+This plugin allows players to transfer one of their luck points to another character. It creates a record of the transaction, including both parties, the date, and the reason given.
+
++============================================================================+
+LUCK GIVEN TO NESSIE (3 LP)
+
+Luck from Joel                                                    2019-07-05
+For a Minor Heal potion.
+
+-----[   page 1 of 1   ]------------------------------------------------------
+
+## Configuration
+
+There is no configuration for this plugin. Although there is a helpfile included, you may wish to remove it and instead add the commands to your primary 'luck' helpfile (/plugins/fs3skills/help/en/luck.md).
+
+## Web Portal
+
+This plugin has no web portal code.  
+
+## Credits
```

Comment:
You should move the credits up near the top so everyone knows who did it!

Reply:
Done.